test/dist/scss.test.js

```diff
@@ -1,35 +0,0 @@
-const path = require('path');
-const { renderSync } = require('sass');
```

Comment:
In retrospect, I probably could have updated this to use the `sass-embedded` package, but this test has been a source of some headaches (mostly slowness failing tests), and I'm not sure how much value it provides.